src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -83,11 +84,10 @@ export const StyledContentLink = styled.a`
 
 // tslint:disable-next-line: variable-name
 const ShowPracticeQuestions = () => {
-  const {book, page} = useSelector(contentSelectors.bookAndPage);
+  const { book, page } = useSelector(contentSelectors.bookAndPage);
```

Comment:
We're trying to avoid unnecessary changes - I usually write this kind of statements with spaces, but there are many places in the code which do not use that styling